### PR TITLE
[functionality] Implement completion() methods #168

### PR DESCRIPTION
**Pull request by C0deXG**
- Implemented completion() method for meta-reference, fireworks, together, ollama, and bedrock providers
   - Each implementation handles both streaming and non-streaming responses
   - Converted single content to messages when necessary
   - Ensured proper yield of CompletionResponse or CompletionResponseStreamChunk objects

### CODE REVIEW

File: llama_stack/providers/impls/meta_reference/inference/inference.py [hunk 1]
```diff
@@ -127,86 +127,4 @@ async def chat_completion(
...
-                    )
-                )
-
-                # TODO(ashwin): what else do we need to send out here when everything finishes?
-            else:
-                yield ChatCompletionResponse(
-                    completion_message=message,
-                    logprobs=logprobs if request.logprobs else None,
-                )
+                    buffer = buffer[len("
```
Comment (ashwinb) [inline llama_stack/providers/impls/meta_reference/inference/inference.py:130]: uh oh, this looks like an unintentional deletion?

**Comment by ashwinb** [comment]
Can you also run `pre-commit`: 

```
cd llama-stack
pip install pre-commit
pre-commit install
```

then -- whenever you run any git commit, it will run the linting hooks.

**Comment by C0deXG** [comment]
> Can you also run `pre-commit`:
> 
> ```
> cd llama-stack
> pip install pre-commit
> pre-commit install
> ```
> 
> then -- whenever you run any git commit, it will run the linting hooks.

yes

File: llama_stack/providers/impls/meta_reference/inference/inference.py (hunk 1 above)
Comment (C0deXG) [reply llama_stack/providers/impls/meta_reference/inference/inference.py:130]: > uh oh, this looks like an unintentional deletion?

in my machine is not cut off i dont know why

**Comment by ashwinb** [comment]
Hey @C0deXG are you still interested and have the bandwidth to follow up on this PR?

**Comment by ashwinb** [comment]
This is certainly stale and no longer necessary.

**Pull request closed without merging**